test/src/e2e_vm_tests/test_programs/should_pass/payable_non_zero_coins/src/main.sw

```diff
@@ -0,0 +1,18 @@
+script;
+
+dep wallet_abi;
+
+use std::constants::ZERO_B256;
+use wallet_abi::Wallet;
+
+fn main() {
+    let contract_address = 0x9299da6c73e6dc03eeabcce242bb347de3f5f56cd1c70926d76526d7ed199b8b;
+    let caller = abi(Wallet, contract_address);
+    // `coins:` is missing (its default value is zero) and hence this is not an error
+    // even that `send_funds` is not payable
```

Comment:
The comment here does not match what the test is doing.

Reply:
Good catch! Fixed in https://github.com/FuelLabs/sway/pull/3450/commits/32be2e8bedbed1b9de7e6cf3b63b9be272781487